exercises/07-test_template.md

```diff
@@ -107,32 +101,33 @@ We can then remove all of our existing test cases from `invalid_login.robot` tes
 
 Our `Settings` table should look like this:
 
-```
+```robot
 *** Settings ***
-Library    SeleniumLibrary
+Library    Browser
 Resource    resource.robot
 Test Setup    Open Browser To Login Page
-Test Teardown    Close Browser
 Test Template    Template Error Page Is Visible When Using Incorrect Credentials
 ```
 
 Let's create our very first data driven test case.
 
-So now that we're using the data driven style of creating test case, under test case table we can write `<name_of_the_test_case> <username> <password>`. Like this:
+So now that we're using the data driven style of creating test case, under test case table we can write
+`<name_of_the_test_case>    <username>    <password>`. Like this:
 
-```
+```robot
 *** Test Cases ***
 #test case name                  #username   #password
```

Comment:
Don't know if necessary, but it could be mentioned here that this line isn't actually needed.

Are comments handled before this?

Reply:
Good point. Added a note about this.